tools/packages.microsoft.com/mapping.json

```diff
@@ -10,6 +10,11 @@
           "distribution" : ["trusty"],
           "PackageFormat" : "PACKAGE_NAME-POWERSHELL_RELEASE-1.rh.x86_64.rpm"
       },
+      {
```

Comment:
Do we need to do this? EOL was 05/17/2022 
https://endoflife.software/operating-systems/linux/fedora